src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -1,115 +1,165 @@
+        assertThat(지하철중복으로만들기_response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
     }
 
     /**
-     * Given 2개의 지하철역을 생성하고
-     * When 지하철역 목록을 조회하면
-     * Then 2개의 지하철역을 응답 받는다
+     * Given 2개의 지하철역을 생성하고 When 지하철역 목록을 조회하면 Then 2개의 지하철역을 응답 받는다
      */
     @DisplayName("지하철역을 조회한다.")
     @Test
     void getStations() {
+        //given
+        지하철역만들기("강남역");
+        지하철역만들기("양재역");
+
+        //when
+        ExtractableResponse<Response> response = 지하철역조회하기();
+
+        //then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertThat(response.jsonPath().getList("$"))
+            .extracting("id")
+            .containsExactly(1, 2);
+        assertThat(response.jsonPath().getList("$"))
+            .extracting("name")
+            .containsExactly("강남역", "양재역");
+    }
+
+    private ExtractableResponse<Response> 지하철역조회하기() {
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().get("/stations")
+            .then().log().all()
+            .extract();
+
+        return response;
+    }
+
+    private ExtractableResponse<Response> 지하철역만들기(String 생성할_지하철역_이름) {
+        Map<String, String> params = new HashMap<>();
+        params.put("name", 생성할_지하철역_이름);
+        return RestAssured.given().log().all()
+            .body(params)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().post("/stations")
+            .then().log().all()
+            .extract();
     }
 
     /**
-     * Given 지하철역을 생성하고
-     * When 그 지하철역을 삭제하면
-     * Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
+     * Given 지하철역을 생성하고 When 그 지하철역을 삭제하면 Then 그 지하철역 목록 조회 시 생성한 역을 찾을 수 없다
      */
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+        지하철역만들기("강남역"); //id 1
+        지하철역만들기("역삼역"); //id 2
+
+        지하철역지우기(1);
+
+        ExtractableResponse<Response> response = 지하철역조회하기();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertThat(response.jsonPath().getList("$"))
+            .extracting("id")
+            .containsExactly(2);
+        assertThat(response.jsonPath().getList("$"))
+            .extracting("name")
+            .containsExactly("역삼역");
+    }
+
+    private ExtractableResponse<Response> 지하철역지우기(long 지하철역_id) {
+        return RestAssured.given().log().all()
+            .body("")
+            .when().delete("/stations/" + 지하철역_id)
+            .then().log().all()
+            .extract();
+    }
+
+    /**
+     * Given 지하철역을 생성하고
+     * When 똑같은 이름의 지하철을 생성하면
+     * Then 생성이 되지 않고 에러(400)이 발생한다     */
+    @Test
+    @DisplayName("중복된 지하철이름을 저장시 에러가 발생한다")
+    public void 지하철중복저장하기(){
+
+        ExtractableResponse<Response> responseOk = 지하철역만들기("강남역");
+        ExtractableResponse<Response> response = 지하철역만들기("강남역");
```

Comment:
`ExtractableResponse<Response> responseOk = 지하철역만들기("강남역");` 가 `given` 이고
해당 `line code` 가 `when` 인 것 같은데, `new line` 없이 작성하니 주석 없이 알아보기 힘들 수 있다는 생각이 드는데 어떻게 생각하시나요?

```suggestion

        ExtractableResponse<Response> response = 지하철역만들기("강남역");
```